baseplate/lib/thrift_pool.py

```diff
@@ -157,6 +162,7 @@ def __init__(
         timeout: float = 1,
         max_connection_attempts: int = 3,
         protocol_factory: TProtocolFactory = _DEFAULT_PROTOCOL_FACTORY,
+        queue: str = "lifo",
```

Comment:
I think it'd be a clearer API and more flexible to take a queue class object here rather than a string. 

```suggestion
        queue: Type = gevent_queue.LifoQueue,
```

Reply:
I thought about that, but my Python is bad and didn't know if that was a recommended way to do things. :smile: